backend/Squirrel.Core/Squirrel.Core.DAL/Context/EntityConfigurations/ProjectDatabaseConfig.cs

```diff
@@ -11,6 +11,8 @@ public void Configure(EntityTypeBuilder<ProjectDatabase> builder)
         builder.Property(x => x.DbName).IsRequired().HasMaxLength(20);
         builder.Property(x => x.Guid).IsRequired();
 
+        builder.HasAlternateKey(x => x.Guid);
```

Comment:
to be a key - it should be one time set-up and read-only (if it is - just resole it)

Reply:
we need it as a key because you can't add a couple dbs to one console app